contracts/modules/staking/StakingPoolFactory.sol

```diff
@@ -13,6 +13,8 @@ contract StakingPoolFactory is IStakingPoolFactory {
   // temporary beacon address storage to avoid constructor arguments in the proxy
   address public beacon;
 
+  address public coverAddress;
```

Comment:
This contract is immutable and cannot be changed or replaced. Revert any changes here as they're neither possible nor required.